sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/Table.java

```diff
@@ -104,8 +104,8 @@ default Map<String, String> properties() {
   default Constraint[] constraints() { return new Constraint[0]; }
 
   /**
-   * Returns the current table version if implementation supports versioning.
+   * Returns this table version without refreshing state if implementation supports versioning.
```

Comment:
Hmm, can you explain more why to specify specially on "without refreshing state"? Does `currentVersion` implicitly refresh state possibly? I suppose `currentVersion` just returns the current version, and don't expect it will refresh state.

Reply:
+1

Reply:
Yeah, the whole purpose of the rename is to make sure connectors DO NOT automatically refresh the underlying table state. In other words, it must act like a guidance for connectors.

Reply:
What about this, folks?

```
/**
 * Returns the version of this table if versioning is supported, null otherwise.
 * <p>
 * This method must not trigger a refresh of the table metadata. It should return
 * the version that corresponds to the current state of this table instance.
 */
```

Reply:
I want to avoid the word `current` as this may not be current and can be behind the version in the metastore.